crane/cmd.go

```diff
@@ -2,9 +2,10 @@ package crane
 
 import (
 	"fmt"
-	"github.com/alecthomas/kingpin"
 	"os"
 	"strings"
+
+	"github.com/alecthomas/kingpin"
```

Comment:
that's the output from goimports - https://godoc.org/golang.org/x/tools/cmd/goimports